### PR TITLE
Add support for Golang Weekly newsletters

### PR DESCRIPTION
**Pull request by sywhb**

### CODE REVIEW

File: packages/inbound-email-handler/src/index.ts [hunk 1]
```diff
@@ -8,16 +8,35 @@ import parseHeaders from 'parse-headers'
...
 import { handlePdfAttachment } from './pdf'
+import { SubstackHandler } from './substack-handler'
+import { AxiosHandler } from './axios-handler'
+import { BloombergHandler } from './bloomberg-handler'
+import { GolangHandler } from './golang-handler'
 
 const NON_NEWSLETTER_EMAIL_TOPIC = 'nonNewsletterEmailReceived'
 const pubsub = new PubSub()
 
+const NEWSLETTER_HANDLERS = [
```
Comment (jacksonh) [inline packages/inbound-email-handler/src/index.ts:24]: Nice

File: packages/inbound-email-handler/src/newsletter.ts [hunk 2]
```diff
@@ -6,12 +6,70 @@ const EMAIL_CONFIRMATION_CODE_RECEIVED_TOPIC = 'emailConfirmationCodeReceived'
...
+      throw new Error('invalid newsletter url')
+    }
+
+    // get author name from email
+    // e.g. 'Jackson Harper from Omnivore App <jacksonh@substack.com>'
+    // or 'Mike Allen <mike@axios.com>'
+    const authors = from.includes(' from ')
+      ? from.split(' from')
+      : from.split(' <')
+    if (!authors) {
```
Comment (jacksonh) [inline packages/inbound-email-handler/src/newsletter.ts:57]: I wonder if one of the mail parsing libraries has some methods for stuff like this.

**Review by jacksonh — APPROVED** [review]

File: packages/inbound-email-handler/src/newsletter.ts (hunk 2 above)
Comment (sywhb) [reply packages/inbound-email-handler/src/newsletter.ts:57]: This one might suit our need: https://www.npmjs.com/package/addressparser

Comment (sywhb) [reply packages/inbound-email-handler/src/newsletter.ts]: I tried https://www.npmjs.com/package/addressparser and it seems okay

**Pull request merged**